src/test/java/org/inferred/freebuilder/processor/ListMultimapPropertyFactoryTest.java

```diff
@@ -628,11 +628,11 @@ public void testMergeFrom_builder() {
         .with(new Processor())
         .with(MULTIMAP_PROPERTY)
         .with(testBuilder()
-            .addLine("DataType.Builder template = DataType.builder()")
+            .addLine("DataType.Builder _template = DataType.builder()")
```

Comment:
Could you revert all the changes to tests that are just using a variable named "template"?